index.js

```diff
@@ -1 +1,99 @@
-// implement your API here
+const express = require('express');
+
+const db = require('./data/db.js');
+
+const server = express();
+
+server.use(express.json());
+
+
+//Post
+server.post('/api/users', (req, res) => {
+    const { name, bio, created_at, updated_at } = req.body;
```

Comment:
```suggestion
    const { name, bio } = req.body;
```
created at and updated at will be auto generated by the database